### PR TITLE
Update docs on how to add your own powershell executables

### PR DESCRIPTION
**Pull request by TylerLeonhardt**
A while ago @rkeithhill added this new feature for adding additional PowerShell exes. This is to document that and also have a place to point people to on how to access the session menu.

I plan on sending a PR to the Azure Functions docs as well pointing this out since folks seem to have a hard time discovering the session menu there.

cc @rjmholt @SydneyhSmith

### CODE REVIEW

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[a231821](https://github.com/MicrosoftDocs/PowerShell-Docs/pull/4312/commits/a231821d3da45355087cfa4efd96a0408def0db7)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/components/vscode/using-vscode.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/add-your-own-powershell-executables/reference/docs-conceptual/components/vscode/using-vscode.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/vscode/using-vscode?view=powershell-3.0&branch=pr-en-us-4312) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/vscode/using-vscode?view=powershell-4.0&branch=pr-en-us-4312) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/vscode/using-vscode?view=powershell-5.0&branch=pr-en-us-4312) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/vscode/using-vscode?view=powershell-5.1&branch=pr-en-us-4312) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/vscode/using-vscode?view=powershell-6&branch=pr-en-us-4312) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C17%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201905171721089601-4312%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=5iV%2BKyOflHHkZYYjL9iZzTBH8cbRYDnju43744jlE0Y%3D&st=2019-05-17T17%3A21%3A35Z&se=2019-06-17T17%3A26%3A35Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[5ed1cdd](https://github.com/MicrosoftDocs/PowerShell-Docs/pull/4312/commits/5ed1cddc74c1ea9ee2cede3438e9d1971b2467a2)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/components/vscode/using-vscode.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/add-your-own-powershell-executables/reference/docs-conceptual/components/vscode/using-vscode.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/vscode/using-vscode?view=powershell-3.0&branch=pr-en-us-4312) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/vscode/using-vscode?view=powershell-4.0&branch=pr-en-us-4312) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/vscode/using-vscode?view=powershell-5.0&branch=pr-en-us-4312) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/vscode/using-vscode?view=powershell-5.1&branch=pr-en-us-4312) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/components/vscode/using-vscode?view=powershell-6&branch=pr-en-us-4312) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C17%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201905171917393010-4312%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=enX91R%2FQFtDTx6PTKSCOlfVZMcr1t0Mqu2tnfQ393yw%3D&st=2019-05-17T19%3A17%3A27Z&se=2019-06-17T19%3A22%3A27Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/docs-conceptual/components/vscode/using-vscode.md [hunk 1]
```diff
@@ -83,27 +83,83 @@ You are prompted with "Do you want to run software from this untrusted publisher
 Type `R` to run the file. Then, open Visual Studio Code and check that the PowerShell extension is
 functioning properly. If you still have issues getting started, let us know on [GitHub](https://github.com/PowerShell/vscode-powershell/issues).
 
-#### Using a specific installed version of PowerShell
+#### Choosing a version of PowerShell to use with the extension
 
-If you wish to use a specific installation of PowerShell with Visual Studio Code, you need to add a new variable to your user settings file.
+With PowerShell Core installing side-by-side with Windows PowerShell, is it now possible to a
```
Comment (rkeithhill) [inline reference/docs-conceptual/components/vscode/using-vscode.md]: Sorry, late to this but aren't we missing the word `use` at the end of this line?

Comment (sdwheeler) [reply reference/docs-conceptual/components/vscode/using-vscode.md]: @rkeithhill Thanks for catching that. I have fixed it now.

**Pull request merged**